themes/custom/ts_wrin/sass/components/_project-detail.scss

```diff
@@ -47,6 +47,8 @@
       img {
         width: 100%;
         height: auto;
+        max-width: 250px;
```

Comment:
Now that I see this, can you move this max-width down to the new rule you've made below? I worry that some of the other rules in this list (`landing-page__content` and `.publication__image`) may not want this max-width rule

Reply:
fixed